catalog/datasets/HadISST.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Would it be a lot of work to change the dataset data to yaml? I feel that would be a tad more user friendly for many folks?

Reply:
it's definitely possible to use YAML as the default format.  i will implement an automation process that aggregates individual YAML files into a single JSON file which can be consumed by the web catalog. This will enable users to work solely with YAML files while benefiting from the convenience resulting from using a unified JSON file as input for  https://leap-data-catalog.vercel.app/